test/spec/index.spec.js

```diff
@@ -14,33 +16,147 @@ describe('index.js', function() {
       var parsedOptions = krabby._getOptions(options);
       assert(_.isEqual(options, parsedOptions));
     });
+    it('finds a krabby config when found inside of another object', function() {
+      var options = {
+        krabby: {
+          tests: [],
+          reports: [],
+          'extra-option': true
+        }
+      };
+
+      var parsedOptions = krabby._getOptions(options);
+      assert(_.isEqual(options.krabby, parsedOptions));
+    });
+    it('throws an Error it can\'t find a config', function() {
+
+      var krabby = proxyquire('../..', {
+        fs: {
+          'existsSync': function() {
+            return;
+          }
+        },
+        'find-parent-dir': {
+          sync: function() {
+            return;
+          }
+        }
+      });
+
+      assert.throws(function() {
+        krabby._getOptions({});
+      }, /can't find the root of the project\. go find a package\.json file/);
+    });
+    it('throws an Error if it can\'t load a valid config', function() {
+
+      // mock out the existsSync to make it seem like a package.json is in
+      // the same dir, but return an empty object for path.join (which gets
+      // sent into `require`). @noCallThru lets proxyquire load a fake module
+      var krabby = proxyquire('../..', {
+        fs: {
+          'existsSync': function(file) {
+            return file === 'package.json';
+          },
+        },
+        path: {
+          join: function() {
+            return '/krabby_test_bad_pkg.json';
+          }
+        },
+        '/krabby_test_bad_pkg.json': {
+          '@noCallThru': true
+        }
+      });
+
+      assert.throws(function() {
+        krabby._getOptions({});
+      }, /no krabby config found\. what do you expect to happen\?/);
+    });
+    it('loads a package.json file from the current dir', function() {
+
+      var krabby = proxyquire('../..', {
+        fs: {
+          'existsSync': function(file) {
+            return file === 'package.json';
+          },
+        },
+        path: {
+          join: function() {
+            return '/krabby_test_pkg.json';
+          }
+        },
+        '/krabby_test_pkg.json': {
+          krabby: {
+            tests: [],
+            reports: []
+          },
+          '@noCallThru': true
+        }
+      });
+
+      assert.doesNotThrow(function() { krabby._getOptions({}); });
```

Comment:
All you're testing is that it doesn't fail catastrophically but maybe it would be better to test for the existence of the config you are expecting.
```
var opts = krabby._getOptions({});
assert(_.isArray(opts.tests));
assert(_.isArray(opts.reports));
```